packages/firestore/src/api/settings.ts

```diff
@@ -88,8 +88,11 @@ export interface FirestoreSettings extends LiteSettings {
    * detect if long-polling should be used. This is very similar to
    * `experimentalForceLongPolling`, but only uses long-polling if required.
    *
-   * This setting will likely be enabled by default in future releases and
-   * cannot be combined with `experimentalForceLongPolling`.
+   * After having had a default value of `false` since its inception 4 years
```

Comment:
Note to self: update this documentation immediately after release to mention the year "2019" instead of "4 years ago" and the actual version number instead of "the most recent release".

Reply:
Googlers see b/278957890 for details.

Reply:
Suggestion: Just say "since 2019" and say "and changed in mid 2023" so you don't need to worry about it later?

Reply:
Done. I still think I want to go back and change it to the specific version and release date so that customers can more easily correlate their issues with the release. But at least with your wording suggestion it's not as time-critical to get that change in.